pkg/json/encode_test.go

```diff
@@ -108,3 +108,25 @@ func TestLength(t *testing.T) {
 		fmt.Sprintf("expected: %s\n", expected)+
 		fmt.Sprintf("actual  : %s\n", actual))
 }
+
+// TODO
```

Comment:
redundant please remove

Reply:
What do you mean, it is not part of the PR?

Reply:
ok